proj/src/components/Header.js

```diff
@@ -1,10 +1,20 @@
 import React from 'react'
 import moment from 'moment'
-export default (props) => {
+import fp from 'lodash/fp'
```

Comment:
import only the functions you want to use. i.e `import { head, last } from lodash/fp`